test/Microsoft.IdentityModel.Tokens.Tests/Microsoft.IdentityModel.Tokens.Tests.csproj

```diff
@@ -22,6 +22,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.KeyVault.Cryptography" Version="$(MicrosoftAzureKeyVaultCryptographyVersion)" />
     <PackageReference Include="System.Text.RegularExpressions" Version="$(SystemTextRegularExpressions)"/>
+    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="$(MicrosoftRestClientRuntimeVersion)"/>
```

Comment:
Was there a mistake in removing this package?

Reply:
I was thinking about the same. Removing the package did not remove the warnings/errors, apparently.

Reply:
We need to specify a certain version MS.Rest.ClientRuntime